crates/swc_ecma_minifier/tests/fixture/next/47005/input.js

```diff
@@ -0,0 +1,539 @@
+(self["webpackChunk_N_E"] = self["webpackChunk_N_E"] || []).push([
```

Comment:
I tried my best to reduce this, but these seem quite close to the minimal required code.